src/sparseml/pytorch/sparsification/quantization/helpers.py

```diff
@@ -481,20 +481,25 @@ def configure_module_default_qconfigs(module: Module):
             submodule.configure_qconfig()
 
 
-def add_quant_dequant(module, name=None, parent_module=None):
+def add_quant_dequant(
+    module: torch.nn.Module, name=None, parent_module=None, layer_class_names=None
+):
     """
     Wraps all Conv and Linear submodule with a qconfig with a QuantWrapper
     :param module: the module to modify
     :param name: name of the module to modify; default to None
     :param parent_module: parent module containing the module to modify; default to None
+    :param layer_class_names: list of module class names to be added to the
+        list of quantizable modules
     :return: the modified module
     """
     named_children = module.named_children()
-    if (
-        type(module) in _QUANTIZABLE_MODULE_TYPES
-        and hasattr(module, "qconfig")
-        and module.qconfig
-    ):
+    is_quantizable = type(module) in _QUANTIZABLE_MODULE_TYPES
+    if layer_class_names:
```

Comment:
Given that this fix is for the new GELUActivation, is it possible to extend the `_QUANTIZABLE_MODULE_TYPES` to include this new module type? This is the exhaustive list of module types that could be quantized, and we should keep it that way if possible.

Reply:
GELUActivation is a not a standard module, but rather a custom module created within HF interface. I'd rather keep is as a recipe input since in other contexts the module can be defined differently

Reply:
OK got it